wftests/warrior_tests/testcases/common_actions_tests/tc_verify_data_regex.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" ?>
```

Comment:
@ryadaval-warrior 
This is a new testcase. Was this added in the related suite?

Reply:
Done. Added.